### ISSUE TITLE
IndexError: pop from empty list

### ISSUE THREAD

**Issue by zchrissirhcz** [vra/easybox#1]
Hi vra

Thanks for providing this easy-to-use label tool.
When I was labeling boxes for one single image and press 'save', I notice the terminal give some error message:

```
E:\projects\vpd\model_differ>easybox
libpng warning: iCCP: cHRM chunk does not match sRGB
Exception in Tkinter callback
Traceback (most recent call last):
  File "e:\soft\miniconda3\lib\tkinter\__init__.py", line 1705, in __call__
    return self.func(*args)
  File "e:\soft\miniconda3\lib\site-packages\easybox\main.py", line 347, in delete_box
    self.bboxes.pop()
IndexError: pop from empty list
```

The version of easybox is 0.0.4 which is the latest for now (2020-08-05 08:49:55).

**Comment by zchrissirhcz** [comment]
To reproduce, 
the image I'm using is:
![000022](https://user-images.githubusercontent.com/3831847/89359633-e32f9000-d6f8-11ea-902d-1c1263215049.jpg)

the system I'm using is Windows 10

the Python version I'm using is 3.7.4

the boxes I'm labeling are:
```
47 321 118 450 3
199 24 284 145 4
32 170 229 249 5
```

**Comment by vra** [comment]
Hi @zchrissirhcz ,
This should be a bug that this peice of code didn't process the boundary condition successfully. I will fix it these two days.  Thanks for pointing it out.

**Comment by vra** [comment]
Hi @zchrissirhcz ,
This  bug shoule has been fixed in latest version 0.05, please update easybox to 0.0.5 to check if everything is okay.

The error is caused by  a mismatch of visual bounding box shown in left and box value shown in right. When the box is too small (e.g., smaller then 2pixel), we should ignore it. I removed box value in right but forgot to remove the visual bounding box in left (although it is too small so we cannot see it), which cause the number of left element and right didn't match (e.g., 4 visual bounding box in left, 3 box value in right).

Thanks again for trying this program.  Please tell me if something is still not okay.

**Comment by zchrissirhcz** [comment]
Tested, this issue has been resolved. Thanks.
